pkg/images/utils/utils_test.go

```diff
@@ -283,3 +283,130 @@ func TestNormalizeImageFullName(t *testing.T) {
 		})
 	}
 }
+
+func TestIsRedHatImageName(t *testing.T) {
```

Comment:
I think this test is redundant

Reply:
How I see it is `TestIsRedHatImageName` tests the logic of whether an image is Red Hat by looking at its registry/remote, while `TestIsRedHatImage` tests how images with several names are handled, making sure we report Red Hat image if it has at least one Red Hat name.

I agree that this could be redundant if we add enough cases to `TestIsRedHatImage`, to cover the different Red Hat `quay.io` remotes, several Red Hat registries etc, but I like the separation of concerns here. So I'd rather leave it as-is, unless there are concerns I'm unaware of.

What do you think?

Reply:
> but I like the separation of concerns here.
Sounds good